docs/guides/step-by-step/step-99.md

```diff
@@ -44,7 +44,7 @@ If that feels like too much possibility to you, why not one of these options:
 -   Share your experience with Netdata and this guide. Be sure to [@mention](https://twitter.com/linuxnetdata) us on 
     Twitter!
 -   Contribute to what we do. Browse our [open issues](https://github.com/netdata/netdata/issues) and check out out
```

Comment:
Not sure why this is broken, but I've noticed that this step is missing in https://github.com/netdata/netdata/blob/fix-broken-links/docs/guides/step-by-step/step-00.md

Reply:
I guess we can't refer to `/README.md` 🤔 Need to check it.